WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -34,7 +34,6 @@ public class CoinViewModel : ViewModelBase, IDisposable
 
 		public CoinListViewModel Owner { get; }
 		private Global Global { get; }
-		public bool InCoinJoinContainer { get; }
```

Comment:
This property is still used here https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml#L87

Reply:
![image](https://user-images.githubusercontent.com/9844978/72611518-a0f45c00-392a-11ea-92cc-657cb94b2816.png)

Reply:
However, I can imagine that this can be a useful feature here. You want to send but the coin is enqueued in CJ right click and just simple dequeue it.

Reply:
Yes, I think you are right, it could be convenient to allow dequeue from SendTab too. I would like to do that but the logic is in the CoinJoinTab and the CoinViewModel knows its owner (and invoke its owner DequeueCoin method) so, it is a bit messy right now and this PR would become big and complex.

I would like to go step by step in the direction of removing that dependency first if possible.

Reply:
ACK, but I wouldn't go into that in this PR. I remember trying, but failing to implement it at one point, not sure why.